### PR TITLE
Added a blog

### PR DESCRIPTION
**Pull request by fish-ball**
Fixes # .

Make sure these boxes are checked before creating your pull request -- thank you!

- [x] Add Blog Header, including title, author, date.
e.g.

``` md
---
 title: "'Score Your Face Photo' a ML&Wechaty practice"
 date: 2017-09-18 09:00 +0800
 author: huyingxi
 ---
 > Author: [@huyingxi](https://github.com/huyingxi/wechaty_selfie) enjoying ML&Wechaty at BUPT
```

- [x] Keep all filenames & url as lowercase, and use `-` to connect words instead of space. e.g. `2017-10-06-wechat-pc-impactor` instead of `2017-10-06-WeChat PC Impactor`,
- [x] Embed the photo & video before publishing, save all external file to the blog `/download/2018` directory.
- [x] Add `<!--more-->` section to just show the abstract in blog homepage.
- [x] Add teaser image for the blog
- [x] Add your self as an author to the author data file at <https://github.com/wechaty/wechaty.github.io/blob/master/_data/authors.yml> with your GitHub username. And please also remember to reference yourself with the Github username in your blog post, so that we can have nice author information besides your blog after we published it!

[See more](https://github.com/wechaty/wechaty.github.io#contribute-guideline)

### CODE REVIEW

File: _posts/2020-05-11-wechaty-subscription-bot.md [hunk 1]
```diff
@@ -0,0 +1,301 @@
+---
+title: "用Wecahty制作订阅机器人(B站)"
+date: 2020-05-11 13:07 +0800
+author: fish-ball
+categories: event
+tags:
+  - wechaty
+header:
+  teaser: /assets/2020/2020-05-11-wechaty-subscription-bot.png
```
Comment (lijiarui) [inline _posts/2020-05-11-wechaty-subscription-bot.md]: please create a new file `subscription-bot` and add this image to the file.

**Review by lijiarui — CHANGES_REQUESTED** [review]
Thanks for the post.

Please change as I request, thank you！

**Comment by lijiarui** [comment]
Also, if you can resolve the conflict, it could be really nice !

File: _posts/2020-05-11-wechaty-subscription-bot.md (hunk 1 above)
Comment (fish-ball) [reply _posts/2020-05-11-wechaty-subscription-bot.md]: It's done, please have a review.

**Comment by lijiarui** [comment]
@fish-ball Thanks for the update, but where is this file: `2020-05-11-wechaty-subscription-bot.png` ? Please commit this image and then I can merge this.

**Comment by lijiarui** [comment]
ping @fish-ball

**Comment by su-chang** [comment]
@fish-ball Could you please finished this blog firstly? And then it will be merged.

File: _posts/2020-05-11-wechaty-subscription-bot.md (hunk 1 above)
Comment (lijiarui) [reply _posts/2020-05-11-wechaty-subscription-bot.md]: Please create a new folder and put these images into the folder. eg ` /assets/2020/wechaty-subscription-bot/teaser.png`

**Review by lijiarui — CHANGES_REQUESTED** [review]

**Comment by lijiarui** [comment]
@fish-ball Please change as I request. Thanks!

**Comment by fish-ball** [comment]
@lijiarui It's ok now, please check again. Sorry for troubling.

File: _posts/2020-05-11-wechaty-subscription-bot.md [hunk 2]
```diff
@@ -0,0 +1,301 @@
+---
+title: "用Wecahty制作订阅机器人(B站)"
+date: 2020-05-11 13:07 +0800
+author: fish-ball
+categories: event
+tags:
+  - wechaty
+header:
+  teaser: /assets/2020/wechaty-subscription-bot/teaser.png
```
Comment (lijiarui) [inline _posts/2020-05-11-wechaty-subscription-bot.md:9]: Good job.

But where is the teaser image? It seems you delete this file...

**Review by lijiarui — CHANGES_REQUESTED** [review]

File: _posts/2020-05-11-wechaty-subscription-bot.md (hunk 2 above)
Comment (fish-ball) [reply _posts/2020-05-11-wechaty-subscription-bot.md:9]: I forgot to add the file before commit. It's done now.

**Review by lijiarui — APPROVED** [review]
Good job!

**Comment by lijiarui** [comment]
Congratulations! 🎉 🎉 🎉

You become a wechaty contributor successfully! 👏 👏 👏

Please:

- [ ]  Add my wechat: ruirui_0914 and send me this pr link. I will invite you into Wechaty Contributor Room
- [ ] Update your profile here: [Contributor List](https://github.com/wechaty/wechaty/wiki/Contributors), then other contributors will know you better!
- [ ]  Join wechaty org by clicking https://github.com/wechaty

image

**Pull request merged**